app/views/org_admin/phases/container.html.erb

```diff
@@ -53,11 +57,18 @@
 
                   <div class="col-sm-6">
                     <div class='text-right text-muted'>
-                      <i class="fa fa-info-circle small"></i>
+                      <% if template.draft? %>
```

Comment:
See comment above. You can test it by:
- Publish your template
- Edit it to force the creation of a new version (these arrows should appear)
- Go back to the templates page and 'unpublished' the template
- Go back in to edit the current version (arrows may not appear due to the `.draft?` check)